app.py

```diff
@@ -433,9 +470,9 @@ def utility():
                     return redirect(url_for("invoice", token=invoice.token))
                 else:
                     error = "failed to create invoice ({})".format(err)
-        return render_template("utility.html", utility=utility, selected_bank_name=bank_desc["name"], status=status, email=email, amount=amount, utility_name=utility_name, values=values, error=error)
+        return render_template("utility.html", utility=utility, selected_bank_name=bank_desc["name"], status=status, email=email, amount=amount, utility_name=utility_name, values=values, error=error, kyc_info=kyc_info, userinfo=userinfo_content, url=url)
     else:
-        return render_template("utility.html", utility=utility, status=STATUS_CREATE, values=werkzeug.MultiDict())
+        return render_template("utility.html", utility=utility, status=STATUS_CREATE, values=werkzeug.MultiDict(), kyc_info=kyc_info, userinfo=userinfo_content, url=url)
```

Comment:
if 'kyc.ok' fails the var 'kyc_info' will be undefined

Reply:
changing to:
```
kyc = bronze_blueprint.session.get('AccountKyc')
    if not kyc.ok:
        return abort(400)
    kyc_info = kyc.json()
    level = kyc_info['level']
    userinfo = bronze_blueprint.session.get('UserInfo')
    if not userinfo.ok:
        return abort(400)
    userinfo_content = userinfo.json()
```

So it would abort if not ok.

Reply:
This changed.